Sphinx error on freshly baked cookie from this template

I just made a new repository from this cookie cutter template, but `make precommit` yields:

```
[gabbard@saga03 saga_tools]$ make precommit
sphinx-build -nWT -b dummy docs docs/_build/html
Running Sphinx v2.3.1
making output directory... done

Traceback (most recent call last):
  File "/nas/gaia/users/gabbard/apps/miniconda3/envs/argument-attachment/lib/python3.6/site-packages/sphinx/cmd/build.py", line 275, in build_main
    args.tags, args.verbosity, args.jobs, args.keep_going)
  File "/nas/gaia/users/gabbard/apps/miniconda3/envs/argument-attachment/lib/python3.6/site-packages/sphinx/application.py", line 267, in __init__
    self.events.emit('config-inited', self.config)
  File "/nas/gaia/users/gabbard/apps/miniconda3/envs/argument-attachment/lib/python3.6/site-packages/sphinx/events.py", line 103, in emit
    results.append(callback(self.app, *args))
  File "/nas/gaia/users/gabbard/apps/miniconda3/envs/argument-attachment/lib/python3.6/site-packages/sphinx/builders/html.py", line 1176, in validate_html_static_path
    logger.warning(__('html_static_path entry %r does not exist'), entry)
  File "/nas/gaia/users/gabbard/apps/miniconda3/envs/argument-attachment/lib/python3.6/logging/__init__.py", line 1642, in warning
    self.log(WARNING, msg, *args, **kwargs)
  File "/nas/gaia/users/gabbard/apps/miniconda3/envs/argument-attachment/lib/python3.6/site-packages/sphinx/util/logging.py", line 124, in log
    super().log(level, msg, *args, **kwargs)
  File "/nas/gaia/users/gabbard/apps/miniconda3/envs/argument-attachment/lib/python3.6/logging/__init__.py", line 1674, in log
    self.logger.log(level, msg, *args, **kwargs)
  File "/nas/gaia/users/gabbard/apps/miniconda3/envs/argument-attachment/lib/python3.6/logging/__init__.py", line 1374, in log
    self._log(level, msg, args, **kwargs)
  File "/nas/gaia/users/gabbard/apps/miniconda3/envs/argument-attachment/lib/python3.6/logging/__init__.py", line 1444, in _log
    self.handle(record)
  File "/nas/gaia/users/gabbard/apps/miniconda3/envs/argument-attachment/lib/python3.6/logging/__init__.py", line 1454, in handle
    self.callHandlers(record)
  File "/nas/gaia/users/gabbard/apps/miniconda3/envs/argument-attachment/lib/python3.6/logging/__init__.py", line 1516, in callHandlers
    hdlr.handle(record)
  File "/nas/gaia/users/gabbard/apps/miniconda3/envs/argument-attachment/lib/python3.6/logging/__init__.py", line 861, in handle
    rv = self.filter(record)
  File "/nas/gaia/users/gabbard/apps/miniconda3/envs/argument-attachment/lib/python3.6/logging/__init__.py", line 720, in filter
    result = f.filter(record)
  File "/nas/gaia/users/gabbard/apps/miniconda3/envs/argument-attachment/lib/python3.6/site-packages/sphinx/util/logging.py", line 406, in filter
    raise SphinxWarning(message)
sphinx.errors.SphinxWarning: html_static_path entry '_static' does not exist

Warning, treated as error:
html_static_path entry '_static' does not exist
make: *** [doc-lint] Error 2
```

--- a/{{cookiecutter.project_slug}}/docs/conf.py
+++ b/{{cookiecutter.project_slug}}/docs/conf.py
@@ -103,7 +103,7 @@ html_theme = "sphinx_rtd_theme"
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = []
+html_static_path = ["_static"]
 
 # Custom sidebar templates, must be a dictionary that maps document names
 # to template names.

--- a/{{cookiecutter.project_slug}}/docs/conf.py
+++ b/{{cookiecutter.project_slug}}/docs/conf.py
@@ -103,7 +103,7 @@ html_theme = "sphinx_rtd_theme"
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ["_static"]
+html_static_path = []
 
 # Custom sidebar templates, must be a dictionary that maps document names
 # to template names.

